vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -74,32 +73,19 @@
     until: target_running_pods.stdout == ""
 
   - name: Pivot objects to target cluster
-    shell: "{{ CLUSTERCTL_PATH }}/clusterctl move --to-kubeconfig /tmp/target.yaml -n {{ NAMESPACE }} -v 10" 
+    shell: "{{ CLUSTERCTL_PATH }}/clusterctl move --to-kubeconfig /tmp/target.yaml -n {{ NAMESPACE }} -v 10"
 
-  
   - name: Check if machines become running.
     shell: |
         kubectl get machines -n {{ NAMESPACE }} -o json | jq -r '[ .items[]
-        | select (.status.phase == "Running" or .status.phase == "running")
+        | select (.status.phase == "Provisioned")
         | .metadata.name ] | length'
     register: provisioned_machines
     environment:
       KUBECONFIG: "/tmp/target.yaml"
     retries: 10
     delay: 20
-    until: provisioned_machines.stdout == "2"
-
-  - name: Check if metal3machines become provisioned.
-    shell: |
-        kubectl get m3m -n {{ NAMESPACE }} -o json | jq -r '[ .items[]
-        | select (.status.ready == "true")
-        | .metadata.name ] | length'
-    register: provisioned_m3m_machines
-    environment:
-      KUBECONFIG: "/tmp/target.yaml"
-    retries: 10
-    delay: 20
-    until: provisioned_m3m_machines.stdout == "2"
+    until: provisioned_machines.stdout == NUMBER_OF_BMH
```

Comment:
why removing the check on metal3machine ?

Reply:
Added comment that it m3machine will be checked once the bug is fixed in CAPM3.